canonical/ubuntu.com
# Fix ubuntu pro contact us form data

## Done

- Fix contact us form data as requested in the issue

## QA

- Fill in /contact-us/form?product=pro with test data, and check that payload is an object and not a json string


## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12651
Closes https://warthogs.atlassian.net/browse/WD-4074
